room/runtime/src/main/java/androidx/room/DatabaseConfiguration.java

```diff
@@ -316,7 +320,64 @@ public DatabaseConfiguration(@NonNull Context context, @Nullable String name,
                 allowMainThreadQueries, journalMode, queryExecutor, transactionExecutor,
                 multiInstanceInvalidation, requireMigration, allowDestructiveMigrationOnDowngrade,
                 migrationNotRequiredFrom, copyFromAssetPath, copyFromFile, copyFromInputStream,
-                null);
+                null, null);
+    }
+
+     /**
```

Comment:
unrealted to this CL but i'm not sure why we are still keeping older configurations.
DatabaseConfiguration is not created by generated code and even though constructor is public, it is `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)` . is this because we are expected to keep api compatibilty even for restricted? 
cc: @danysantiago

Reply:
Yes - It possible the API tools forced me to start doing this some time ago and know they are able to track restricted APIs and more importantly allow us to be incompatible knowing they are restricted but I have not tried. =/